Makefile

```diff
@@ -110,9 +110,15 @@ delete: check_python check_branch
 index: check_python check_branch
 	python scripts/reindex.py --index --partition-prefix-length=2
 
+reindex_args = --delete --index --purge --partition-prefix-length=2
+
 .PHONY: reindex
 reindex: check_python check_branch
-	python scripts/reindex.py --delete --index --purge --partition-prefix-length=2
+	python scripts/reindex.py ${reindex_args}
+
+.PHONY: small_reindex
+small_reindex: check_python check_branch
```

Comment:
Consider a name that starts with `reindex_` which follows the same pattern used by `make requirements*`.

Reply:
Good catch, Hannes specifically wanted a target that worked with tab-completion.